### ISSUE TITLE
Development environment question

### ISSUE THREAD

**Issue by alograg** [rundeck/rundeck#7639]
Issue type: question

Hello everyone
I'm new to developing in Java and Groovy, I don't know much about the structure and operation of Rundeck either.
I'm trying to correct problem #5987 .
I followed the indications of the [environment](https://github.com/rundeck/rundeck/wiki/IDE-Development-Environment) and [development](https://github.com/rundeck/rundeck/wiki/Building-and-Testing) guide, but I have only been able to debug the groovy part of rundeckapp/grails-app folder. And even so I have not been able to change the behavior to dianic update, of realms, as the [manual says](https://docs.rundeck.com/docs/administration/security/authentication.html#hot-reloading-the-realm-properties-file) for development environment.
Do you have any document/tuto/references on how to debug the jobs section in java?

Thanks for your help

**My Rundeck detail**

* Rundeck version: 3.2.3
* install type: development
* OS Name/version: Ubuntu 20.04.4 LTS
* DB Type/version: mysql
* Realm: file
* IDE: IntelliJ IDEA 2021.3.3
* Compiler: Javac 1.8

**Comment by fdevans** [comment]
Is the issue still happening on current versions of Rundeck?  The Wiki section of the repo has some information about setting up a development environment.

**Comment by alograg** [comment]
Yes, I followed the wiki guides and they don't have debugging instructions.
I don't know if still happen on current versions, i want fixed in that version for learning purposes and take part in the wiki
Meaby @gschueler , @jtobard  or @sjrd218 can help me ?

**Comment by leandroidvlab** [comment]
Same to me, i tried to run a debug with Intellij IDEA Ultimate with Rundeck 4.8.0, but the debug console exit with the this message:

![image](https://user-images.githubusercontent.com/99025208/213268457-fecd190d-ecfe-4845-bc55-9cc2dc44e2e2.png)

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
